heka/sandbox/decoders/extract_telemetry_dimensions.lua

```diff
@@ -230,6 +230,15 @@ local function process_json(msg, json, parsed)
         update_field(msg.Fields, "normalizedChannel" , fx.normalize_channel(auc))
 
         -- The "telemetryEnabled" flag does not apply to this type of ping.
```

Comment:
This comment is to explain the absence of a following code block? (looks like bad comment spacing but probably isn't).